### PR TITLE
small improvement to tests

### PR DESCRIPTION
**Pull request by charlienegri**
## Change Summary
small tweaks to tests to better represent the code after #1128 was merged

## Checklist

* [x] Start with a draft-PR
* [x] The PR title is a good summary of the changes
* [ ] PR is set to AeroTools and a tentative milestone
* [x] Documentation reflects the changes where applicable
* [x] Tests for the changes exist where applicable
* [x] Tests pass locally
* [x] Tests pass on CI
* [x] At least 1 reviewer is selected
* [x] Make PR ready to review

### CODE REVIEW

**Comment by charlienegri** [comment]
the checks with exception raise in _check_statistics_periods and check_if_year are overlapping... the first 2 ifs could be removed in the latter probably, both functions are called only in setupclasses.py and close to each other:
```
        self.time_cfg.periods = _check_statistics_periods(periods)
        start, stop = _get_min_max_year_periods(periods)
        start_yr = start.year
        stop_yr = stop.year
        years = check_if_year(periods)
```

File: hoqq
Comment (lewisblake) [inline hoqq:1]: This file is no longer in main-dev. please merge from main-dev.

**Review by lewisblake — CHANGES_REQUESTED** [review]
Merge main-dev into this to get rid off the hoqq file first, but otherwise it looks good. Thanks for these improved tests!

File: hoqq
Comment (charlienegri) [reply hoqq:1]: rebased thanks to Magnus' advice

**Review by lewisblake — APPROVED** [review]
:rocket:

**Pull request merged**